docs/glossary.rst

```diff
@@ -32,24 +32,23 @@ Glossary
        For more information about the permission system read
        :doc:`api/authentication_api`.
 
-   userid:
+   userid
       The unique id for one userique id of one :term:`group`: "group:<name>".
 
    group
-       A set of users. Can be mapped to permission :term:`role`s.
+       A set of users. Can be mapped to permision :term:`role`s.
```

Comment:
This looks like a regression too. Not sure about same of the other changes in the file...